docs/api-guide/views.md

```diff
@@ -217,11 +217,26 @@ You may pass `None` in order to exclude the view from schema generation.
     def view(request):
         return Response({"message": "Will not appear in schema!"})
 
+# Async Views
+
+When using Django 4.1 and above, REST framework allows you to work with async class and function based views.
+
+For class based views, all handler methods must be async, otherwise Django will raise an exception. For function based views, the function itself must be async.
+
+For example:
+
+    class ListUsers(APIView):
+        async def get(self, request):
+            return Response({"message": "This is an async class based view."})
+
+
+    @api_view(['GET'])
+    async def view(request):
+        return Response({"message": "This is an async function based view."})
```

Comment:
```suggestion
    class AsyncView(APIView):
        async def get(self, request):
            return Response({"message": "This is an async class based view."})


    @api_view(['GET'])
    async def async_view(request):
        return Response({"message": "This is an async function based view."})
```
I suggest using generic names for the class and function so that it is clear that they are examples of equivalent implementations of an asynchronous views